tools/cbmc/proofs/make_common_makefile.py

```diff
@@ -195,16 +195,16 @@ def write_cbmcbatchyaml_target(opsys, _makefile):
 # Build configuration file to run cbmc under cbmc-batch in CI
 
 define encode_options
-'=$(shell echo $(1) | sed 's/ ,/ /g' | sed 's/ /;/g')='
+	'=$(shell echo $(1) | sed 's/ ,/ /g' | sed 's/ /;/g')='
 endef
 
 cbmc-batch.yaml:
 	@echo "Building $@"
 	@$(RM) $@
 	@echo "jobos: ubuntu16" >> $@
-	@echo "cbmcflags: $(call encode_options,$(CBMCFLAGS) --unwinding-assertions)" >> $@
+	@echo "cbmcflags: $(call encode_options,$(CBMCFLAGS))" >> $@
```

Comment:
I think unwinding-assertions should always be checked. Is there a specific reason to remove this flag? At least I would expect it to appear than in the CBMCFLAGS variable.

Reply:
Reverted to master.